specification/network/resource-manager/Microsoft.Network/stable/2019-12-01/virtualNetworkGateway.json

```diff
@@ -2898,6 +2907,10 @@
           "type": "boolean",
           "description": "EnableBgp flag."
         },
+        "useLocalAzureIpAddress": {
```

Comment:
`useLocalAzureIpAddress` and `enablePrivateIpAddress` sound very similar to me and could be confusing to customers. Are there any more descriptive names that could help differentiate these two settings?

Reply:
These setting are on two different resources, and makes sense accordingly.

Reply:
Ah, I missed that. Thanks for clarifying!